frontend/utility/AdvancedOutput.cpp

```diff
@@ -200,7 +200,22 @@ void AdvancedOutput::UpdateStreamSettings()
 		int keyint_sec = (int)obs_data_get_int(settings, "keyint_sec");
 		obs_service_apply_encoder_settings(main->GetService(), settings, nullptr);
 		if (!enforceBitrate) {
-			blog(LOG_INFO, "User is ignoring service bitrate limits.");
+			int maxVideoBitrate;
+			int maxAudioBitrate;
+			obs_service_get_max_bitrate(main->GetService(), &maxVideoBitrate, &maxAudioBitrate);
+
+			std::string videoBitRateLogString = maxVideoBitrate > 0 ? std::to_string(maxVideoBitrate)
+										: "None";
+			std::string audioBitRateLogString = maxAudioBitrate > 0 ? std::to_string(maxAudioBitrate)
+										: "None";
+
+			blog(LOG_INFO,
+			     "User is ignoring service bitrate limits.\n"
+			     "Service Recommendations:\n"
+			     "\tvideo bitrate: %s\n"
+			     "\taudio bitrate: %s",
+			     videoBitRateLogString.c_str(), audioBitRateLogString.c_str());
```

Comment:
I wonder a bit if we're not overcomplicating this. We could instead:
```cpp

			blog(LOG_INFO,
			     "User is ignoring service bitrate limits.\n"
			     "Service Recommendations:\n"
			     "\tvideo bitrate: %d\n"
			     "\taudio bitrate: %d",
			     maxVideoBitrate, maxAudioBitrate);
```

Or are `maxVideoBitrate` and `maxAudioBitrate` not guaranteed to be zero if no value is specified? Is there value in distinguishing `0` from `None`?

Reply:
I'm willing to wait a bit for a other opinions. It just jumped out to me at first glance that it seemed like extra work here, and I wasn't sure how much extra clarity it granted. cc @Warchamp7

Reply:
Yeah I agree that printing a `0` here would seem more like a mistake than an intended value. Being able to interpret that zero as "no recommendation" requires internal magic knowledge about the implementation that a user should not have.

And for better or worse our INFO level log output is watched and parsed by our users (and not just the log-analyser) so it has to be "human-friendly" (which in turn makes the log-parser work harder).

The holes we dig ourselves into.. 😅

Reply:
As I have no other idea to simplify this, I'll resolve this feedback then.